anitya/lib/model.py

```diff
@@ -20,17 +20,28 @@
 anitya mapping of python classes to Database Tables.
 """
 
+try:
+    # The Python 3.6+ API
+    from secrets import choice as random_choice
+except ImportError:
+    from random import choice as random_choice
```

Comment:
dear python, pls to follow semantic versioning. kthxbye

Reply:
``random.choice`` is still present in 3.6 so it actually does.

Reply:
Ah, then I stand happily corrected.